projects/plugins/jetpack/extensions/blocks/contact-form/util/with-style-variables.js

```diff
@@ -0,0 +1,10 @@
+import './form-styles.js';
+import { useCallback } from '@wordpress/element';
+
+export const withStyleVariables = WrappedComponent => props => {
+	const EDITOR_SELECTOR = '[data-type="jetpack/contact-form"]';
+	/* eslint-disable-next-line no-undef */
+	const generateVariables = useCallback( () => generateStyleVariables( EDITOR_SELECTOR ), [] );
```

Comment:
Hmm, is there any particular reason why we need `useCallback` here or is it just to make the linter happy? Like, right now I see little difference between this or calling `generateStyleVariables( EDITOR_SELECTOR )` directly.

Reply:
That would be one of my questions, but I forgot to add it to the PR 😅 
Glad you mention it. I'll remove the `useCallback`